src/harness/Chart.yaml

```diff
@@ -14,7 +14,7 @@ type: application
 # to the chart and its templates, including the app version.
 # Versions are expected to follow Semantic Versioning (https://semver.org/)
 
-version: 0.4.7
+version: 0.4.6
```

Comment:
.8 ?

Reply:
we dont want to release it, it should be okay